org.nxengine.nxengine_evo.json

```diff
@@ -0,0 +1,49 @@
+{
+	"app-id": "org.nxengine.nxengine_evo",
```

Comment:
the `nexengine.org` doesn't resolve. Is this under the control of the project developers? If not then the app-id doesn't meet the requirements.